src/Core/Resolvers/SqlPaginationUtil.cs

```diff
@@ -267,20 +270,20 @@ protected class NextLinkColumn
             RuntimeConfigProvider runtimeConfigProvider
             )
         {
-            List<PaginationColumn>? after = new();
-            IEnumerable<NextLinkColumn>? requestAfter;
+            List<PaginationColumn>? paginationCursorColumnsForQuery = new();
+            IEnumerable<NextLinkColumn>? paginationCursorColumnsFromRequest;
```

Comment:
```suggestion
            IEnumerable<NextLinkColumn>? paginationCursorFieldsFromRequest;
```

Reply:
see comment about naming: 'fields' refer to values received from user/request and 'columns' is a DAB internal reference to actual database metadata. This will also help with readability: knowing what collection holds data directly provided by request versus values internally resolved/translated by DAB into database metdata.